ui/src/app/devices/views/DeviceDetails/DeviceDetailsHeader/DeviceDetailsHeader.test.tsx

```diff
@@ -105,4 +105,20 @@ describe("DeviceDetailsHeader", () => {
       "Delete"
     );
   });
+
+  it("displays the device name if an action is not selected", () => {
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter>
+          <DeviceDetailsHeader
+            headerContent={null}
+            setHeaderContent={jest.fn()}
+            systemId="abc123"
+          />
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find("DeviceName").exists()).toBe(true);
```

Comment:
We already know that selecting by component name won't work with `testing-library`. Could you use a `testid` instead to make future migration easier?

Looks like `data-test` is used in other tests in this file, so would make sense for consistency.

Reply:
You can find guidance to this problem in the [enzyme to RTL doc](https://docs.google.com/document/d/1ysVlPByzVY-NBUjGic1WupHojUl30P6cf3Xe-OVRRPg/edit#heading=h.isw7pahd7mz6) feel free to update this doc(or add comments) accordingly if you find issues